plugins/redfish/fu-plugin-redfish.c

```diff
@@ -221,14 +220,28 @@ fu_redfish_plugin_ipmi_create_user(FuPlugin *plugin, GError **error)
 	if (locker == NULL)
 		return FALSE;
 
-	/* check the slot is clear */
-	username = fu_ipmi_device_get_user_password(device, 0x04, NULL);
-	if (username != NULL) {
-		g_set_error(error,
-			    FWUPD_ERROR,
-			    FWUPD_ERROR_NOT_SUPPORTED,
-			    "cannot create fwupd user with account %s already existing",
-			    username);
+	/* check for existing user, and if not then remember the first spare slot */
+	for (guint8 i = 3; i < 63; i++) {
```

Comment:
these numbers seem somewhat arbitrary and hardcoded, are they implementation specific or speced somewhere?

Reply:
That's an excellent question. Will ask the OEM.

Reply:
Okay, so 0 is reserved, and 1 is typically the factory-set admin account, so 2 is more appropriate. The field "on the wire" is one byte, so 0xFF is a more appropriate maximum. I've fixed up the commit with those changes and tested on real hardware.